blog/models.py

```diff
@@ -25,3 +26,28 @@ class Meta:
         ordering = ['-added']
 
 
+
+class Comment(models.Model):
+    author = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE,
+                             related_name='comments')
+    name = models.CharField(max_length=80)
+    # email = models.EmailField()
+    body = models.TextField()
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+    active = models.BooleanField(default =True)
+
+    class Meta:
+        ordering = ('created',)
+
+    def __str__(self):
+        return 'Comment by {} on {}'.format(self.name, self.post)
+
+class Image(models.Model):
+    title = models.CharField(max_length=200)
+    image = models.ImageField(upload_to='images/')
```

Comment:
Same: does this have to link back to post (using a foreign key field)?

Reply:
Local file storage is not going to work in the cloud (if we go with Heroku for deployment). 

Remind me to show you the `pybites-tools` package so we can reuse some code to upload the images to an AWS S3 bucket (do you have an account) to work around this, ok?